src/main/java/org/jenkinsci/plugins/workflow/cps/steps/LoadFromClasspathStepExecution.java

```diff
@@ -0,0 +1,134 @@
+package org.jenkinsci.plugins.workflow.cps.steps;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Scanner;
+
+import org.apache.tools.ant.taskdefs.Classloader;
+import org.codehaus.groovy.control.MultipleCompilationErrorsException;
+import org.jenkinsci.plugins.workflow.cps.CpsCompilationErrorsException;
+import org.jenkinsci.plugins.workflow.cps.CpsFlowExecution;
+import org.jenkinsci.plugins.workflow.cps.CpsStepContext;
+import org.jenkinsci.plugins.workflow.cps.CpsThread;
+import org.jenkinsci.plugins.workflow.cps.replay.ReplayAction;
+import org.jenkinsci.plugins.workflow.steps.BodyExecutionCallback;
+import org.jenkinsci.plugins.workflow.steps.StepContext;
+import org.jenkinsci.plugins.workflow.steps.StepExecution;
+
+import groovy.lang.Script;
+import hudson.model.TaskListener;
+
+/**
+ * Execution implementation for {@link LoadFromClasspathStep}.
+ *
+ * @author Austin Witt
+ * 
+ * @since 2.74
+ *
+ */
+public class LoadFromClasspathStepExecution extends StepExecution {
+	
+	private static final long serialVersionUID = 1L;
+
+	protected String path;
+
+	protected Object classloaderSource;
+	
+	protected boolean unexport;
+	
+	protected transient TaskListener listener;
```

Comment:
If this is not `transient`, this fails to serialize:

```
an exception which occurred:
	in field org.jenkinsci.plugins.workflow.log.BufferedBuildListener$Replacement.ros
	in object org.jenkinsci.plugins.workflow.log.BufferedBuildListener$Replacement@4a3299ab
	in field org.jenkinsci.plugins.workflow.log.TaskListenerDecorator$CloseableTaskListener.closeDelegate
	in object org.jenkinsci.plugins.workflow.log.TaskListenerDecorator$CloseableTaskListener@33aa7b11
	in field org.jenkinsci.plugins.workflow.cps.steps.LoadFromClasspathStepExecution.listener
	in object org.jenkinsci.plugins.workflow.cps.steps.LoadFromClasspathStepExecution@3428f778
	in field org.jenkinsci.plugins.workflow.cps.CpsThread.step
	in object org.jenkinsci.plugins.workflow.cps.CpsThread@56b269e8
	in field org.jenkinsci.plugins.workflow.cps.CpsThreadGroup.threads
	in object org.jenkinsci.plugins.workflow.cps.CpsThreadGroup@4bfe79d1
	in object org.jenkinsci.plugins.workflow.cps.CpsThreadGroup@4bfe79d1
Caused: java.io.NotSerializableException: The calling thread Thread[Running CpsFlowExecution[Owner[devtools/test-project-please-ignore/master/10:devtools/test-project-please-ignore/master #10]],5,main] has no associated channel. The current object hudson.remoting.RemoteOutputStream@19de90ba is interface org.jenkinsci.remoting.SerializableOnlyOverRemoting, but it is likely being serialized/deserialized without the channel
	at org.jenkinsci.remoting.SerializableOnlyOverRemoting.getChannelForSerialization(SerializableOnlyOverRemoting.java:67)
	at hudson.remoting.RemoteOutputStream.writeObject(RemoteOutputStream.java:83)
```

If this _is_ `transient`, SpotBugs complains: 

```
[2019-08-30T16:34:29.442Z] [ERROR] The field org.jenkinsci.plugins.workflow.cps.steps.LoadFromClasspathStepExecution.listener is transient but isn't set by deserialization [org.jenkinsci.plugins.workflow.cps.steps.LoadFromClasspathStepExecution] In LoadFromClasspathStepExecution.java SE_TRANSIENT_FIELD_NOT_RESTORED
```

I'm okay agreeing with SpotBugs' assessment, but... the [`LoadStepExecution`](https://github.com/jenkinsci/workflow-cps-plugin/blob/master/src/main/java/org/jenkinsci/plugins/workflow/cps/steps/LoadStepExecution.java#L30) has a `transient TaskListener` that doesn't show up in the SpotBugs warnings.... and as far as I can tell, doesn't explicitly re-set the `listener` when it's deserialized.

Have I missed something here? What's the right thing to do with this listener?